src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -19,11 +19,24 @@
 
     @Override
     public CinemaHall get(Long id) {
-        return cinemaHallDao.get(id).get();
+        return cinemaHallDao.get(id).orElseThrow(
+                () -> new RuntimeException("cinemaHall not found"));
     }
 
     @Override
     public List<CinemaHall> getAll() {
         return cinemaHallDao.getAll();
     }
+
+    @Override
+    public Boolean update(CinemaHall cinemaHall) {
+        CinemaHall currentCinemaHall = get(cinemaHall.getId());
+        cinemaHallDao.update(cinemaHall);
+        return currentCinemaHall.equals(get(cinemaHall.getId()));
+    }
+
+    @Override
+    public Boolean delete(Long id) {
```

Comment:
Consider changing the return type of the `delete` method to `void`. Returning a boolean might not be necessary if the method is expected to always succeed or throw an exception otherwise.